### PR TITLE
Enhanced update-ci script

### PR DESCRIPTION
**Pull request by devguyio**
Refactor update-ci script to use the bash TUI functions introduced in #1496 . 

### Output
[asciicast](https://asciinema.org/a/450711)

Should be merged after #1496 
/hold

### CODE REVIEW

File: openshift/ci-operator/update-ci.sh [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # A script that will update the mapping file in github.com/openshift/release
 
 set -e
+
+source "$(dirname "$0")/../common.sh"
```
Comment (matzew) [inline openshift/ci-operator/update-ci.sh:6]: `common.sh` is that a good name for the fancy ui? :smile:

Comment (devguyio) [reply openshift/ci-operator/update-ci.sh:6]: well it's just starting a common script that can include other functions, a convention been followed all over. I hardly see any "specific" sourced shell scripts . Check e2e-common etc. 

I've no strong opinion there, just saying it's same as we've been doing so far.

**Comment by devguyio** [comment]
@matzew @pierDipi ready to be merged
/unhold

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**